.travis.yml

```diff
@@ -65,6 +69,14 @@ script:
   - if [ "$GOOS" = "linux" ]; then sudo PATH=$PATH GOPATH=$GOPATH make integration ; fi
   # Run the integration suite a second time. See discussion in github.com/containerd/containerd/pull/1759
   - if [ "$GOOS" = "linux" ]; then sudo PATH=$PATH GOPATH=$GOPATH TESTFLAGS_PARALLEL=1 make integration ; fi
+  - if [ "$GOOS" = "linux" ]; then
+      sudo PATH=$PATH containerd -log-level debug &> /tmp/containerd-cri.log &
```

Comment:
Why the redirect?

Reply:
It seems that for containerd CI, we only print containerd log when test fails.
We are doing something similar here.